[2022-01-27 15:57:20] local.ERROR: Uncaught Error: Class 'Auth0\Login\Auth0User' not found in D:\Projects\foodtiger\app\Models\MyAuth0User.php:8
Stack trace:
#0 D:\Projects\foodtiger\vendor\composer\ClassLoader.php(444): include()
#1 D:\Projects\foodtiger\vendor\composer\ClassLoader.php(322): Composer\Autoload\includeFile('D:\\Projects\\foo...')
#2 [internal function]: Composer\Autoload\ClassLoader->loadClass('App\\Models\\MyAu...')
#3 [internal function]: spl_autoload_call('App\\Models\\MyAu...')
#4 Command line code(1): class_exists('App\\Models\\MyAu...')
#5 [internal function]: {closure}('App\\Models\\MyAu...')
#6 Command line code(1): array_filter(Array, Object(Closure))
#7 {main}
  thrown {"exception":"[object] (Symfony\\Component\\ErrorHandler\\Error\\FatalError(code: 0): Uncaught Error: Class 'Auth0\\Login\\Auth0User' not found in D:\\Projects\\foodtiger\\app\\Models\\MyAuth0User.php:8
Stack trace:
#0 D:\\Projects\\foodtiger\\vendor\\composer\\ClassLoader.php(444): include()
#1 D:\\Projects\\foodtiger\\vendor\\composer\\ClassLoader.php(322): Composer\\Autoload\\includeFile('D:\\\\Projects\\\\foo...')
#2 [internal function]: Composer\\Autoload\\ClassLoader->loadClass('App\\\\Models\\\\MyAu...')
#3 [internal function]: spl_autoload_call('App\\\\Models\\\\MyAu...')
#4 Command line code(1): class_exists('App\\\\Models\\\\MyAu...')
#5 [internal function]: {closure}('App\\\\Models\\\\MyAu...')
#6 Command line code(1): array_filter(Array, Object(Closure))
#7 {main}
  thrown at D:\\Projects\\foodtiger\\app\\Models\\MyAuth0User.php:8)
[stacktrace]
#0 {main}
"} 
[2022-01-27 15:57:20] local.ERROR: Uncaught Error: Class 'Auth0\Login\Auth0User' not found in D:\Projects\foodtiger\app\Models\MyAuth0User.php:8
Stack trace:
#0 D:\Projects\foodtiger\vendor\composer\ClassLoader.php(444): include()
#1 D:\Projects\foodtiger\vendor\composer\ClassLoader.php(322): Composer\Autoload\includeFile('D:\\Projects\\foo...')
#2 [internal function]: Composer\Autoload\ClassLoader->loadClass('App\\Models\\MyAu...')
#3 [internal function]: spl_autoload_call('App\\Models\\MyAu...')
#4 Command line code(1): class_exists('App\\Models\\MyAu...')
#5 [internal function]: {closure}('App\\Models\\MyAu...')
#6 Command line code(1): array_filter(Array, Object(Closure))
#7 {main}
  thrown {"exception":"[object] (Symfony\\Component\\ErrorHandler\\Error\\FatalError(code: 0): Uncaught Error: Class 'Auth0\\Login\\Auth0User' not found in D:\\Projects\\foodtiger\\app\\Models\\MyAuth0User.php:8
Stack trace:
#0 D:\\Projects\\foodtiger\\vendor\\composer\\ClassLoader.php(444): include()
#1 D:\\Projects\\foodtiger\\vendor\\composer\\ClassLoader.php(322): Composer\\Autoload\\includeFile('D:\\\\Projects\\\\foo...')
#2 [internal function]: Composer\\Autoload\\ClassLoader->loadClass('App\\\\Models\\\\MyAu...')
#3 [internal function]: spl_autoload_call('App\\\\Models\\\\MyAu...')
#4 Command line code(1): class_exists('App\\\\Models\\\\MyAu...')
#5 [internal function]: {closure}('App\\\\Models\\\\MyAu...')
#6 Command line code(1): array_filter(Array, Object(Closure))
#7 {main}
  thrown at D:\\Projects\\foodtiger\\app\\Models\\MyAuth0User.php:8)
[stacktrace]
#0 {main}
"} 
[2022-01-27 15:57:20] local.ERROR: Uncaught Error: Class 'Auth0\Login\Auth0User' not found in D:\Projects\foodtiger\app\Models\MyAuth0User.php:8
Stack trace:
#0 D:\Projects\foodtiger\vendor\composer\ClassLoader.php(444): include()
#1 D:\Projects\foodtiger\vendor\composer\ClassLoader.php(322): Composer\Autoload\includeFile('D:\\Projects\\foo...')
#2 [internal function]: Composer\Autoload\ClassLoader->loadClass('App\\Models\\MyAu...')
#3 [internal function]: spl_autoload_call('App\\Models\\MyAu...')
#4 Command line code(1): class_exists('App\\Models\\MyAu...')
#5 [internal function]: {closure}('App\\Models\\MyAu...')
#6 Command line code(1): array_filter(Array, Object(Closure))
#7 {main}
  thrown {"exception":"[object] (Symfony\\Component\\ErrorHandler\\Error\\FatalError(code: 0): Uncaught Error: Class 'Auth0\\Login\\Auth0User' not found in D:\\Projects\\foodtiger\\app\\Models\\MyAuth0User.php:8
Stack trace:
#0 D:\\Projects\\foodtiger\\vendor\\composer\\ClassLoader.php(444): include()
#1 D:\\Projects\\foodtiger\\vendor\\composer\\ClassLoader.php(322): Composer\\Autoload\\includeFile('D:\\\\Projects\\\\foo...')
#2 [internal function]: Composer\\Autoload\\ClassLoader->loadClass('App\\\\Models\\\\MyAu...')
#3 [internal function]: spl_autoload_call('App\\\\Models\\\\MyAu...')
#4 Command line code(1): class_exists('App\\\\Models\\\\MyAu...')
#5 [internal function]: {closure}('App\\\\Models\\\\MyAu...')
#6 Command line code(1): array_filter(Array, Object(Closure))
#7 {main}
  thrown at D:\\Projects\\foodtiger\\app\\Models\\MyAuth0User.php:8)
[stacktrace]
#0 {main}
"} 
[2022-01-27 15:57:20] local.ERROR: Uncaught Error: Class 'Auth0\Login\Auth0User' not found in D:\Projects\foodtiger\app\Models\MyAuth0User.php:8
Stack trace:
#0 D:\Projects\foodtiger\vendor\composer\ClassLoader.php(444): include()
#1 D:\Projects\foodtiger\vendor\composer\ClassLoader.php(322): Composer\Autoload\includeFile('D:\\Projects\\foo...')
#2 [internal function]: Composer\Autoload\ClassLoader->loadClass('App\\Models\\MyAu...')
#3 [internal function]: spl_autoload_call('App\\Models\\MyAu...')
#4 Command line code(1): class_exists('App\\Models\\MyAu...')
#5 [internal function]: {closure}('App\\Models\\MyAu...')
#6 Command line code(1): array_filter(Array, Object(Closure))
#7 {main}
  thrown {"exception":"[object] (Symfony\\Component\\ErrorHandler\\Error\\FatalError(code: 0): Uncaught Error: Class 'Auth0\\Login\\Auth0User' not found in D:\\Projects\\foodtiger\\app\\Models\\MyAuth0User.php:8
Stack trace:
#0 D:\\Projects\\foodtiger\\vendor\\composer\\ClassLoader.php(444): include()
#1 D:\\Projects\\foodtiger\\vendor\\composer\\ClassLoader.php(322): Composer\\Autoload\\includeFile('D:\\\\Projects\\\\foo...')
#2 [internal function]: Composer\\Autoload\\ClassLoader->loadClass('App\\\\Models\\\\MyAu...')
#3 [internal function]: spl_autoload_call('App\\\\Models\\\\MyAu...')
#4 Command line code(1): class_exists('App\\\\Models\\\\MyAu...')
#5 [internal function]: {closure}('App\\\\Models\\\\MyAu...')
#6 Command line code(1): array_filter(Array, Object(Closure))
#7 {main}
  thrown at D:\\Projects\\foodtiger\\app\\Models\\MyAuth0User.php:8)
[stacktrace]
#0 {main}
"} 
[2022-01-27 15:57:20] local.ERROR: Uncaught Error: Class 'Auth0\Login\Auth0User' not found in D:\Projects\foodtiger\app\Models\MyAuth0User.php:8
Stack trace:
#0 Command line code(1): include_once()
#1 {main}
  thrown {"exception":"[object] (Symfony\\Component\\ErrorHandler\\Error\\FatalError(code: 0): Uncaught Error: Class 'Auth0\\Login\\Auth0User' not found in D:\\Projects\\foodtiger\\app\\Models\\MyAuth0User.php:8
Stack trace:
#0 Command line code(1): include_once()
#1 {main}
  thrown at D:\\Projects\\foodtiger\\app\\Models\\MyAuth0User.php:8)
[stacktrace]
#0 {main}
"} 
[2022-01-27 15:58:01] local.ERROR: Uncaught Error: Class 'Auth0\Login\Auth0User' not found in D:\Projects\foodtiger\app\Models\MyAuth0User.php:8
Stack trace:
#0 D:\Projects\foodtiger\vendor\composer\ClassLoader.php(444): include()
#1 D:\Projects\foodtiger\vendor\composer\ClassLoader.php(322): Composer\Autoload\includeFile('D:\\Projects\\foo...')
#2 [internal function]: Composer\Autoload\ClassLoader->loadClass('App\\Models\\MyAu...')
#3 [internal function]: spl_autoload_call('App\\Models\\MyAu...')
#4 Command line code(1): class_exists('App\\Models\\MyAu...')
#5 [internal function]: {closure}('App\\Models\\MyAu...')
#6 Command line code(1): array_filter(Array, Object(Closure))
#7 {main}
  thrown {"exception":"[object] (Symfony\\Component\\ErrorHandler\\Error\\FatalError(code: 0): Uncaught Error: Class 'Auth0\\Login\\Auth0User' not found in D:\\Projects\\foodtiger\\app\\Models\\MyAuth0User.php:8
Stack trace:
#0 D:\\Projects\\foodtiger\\vendor\\composer\\ClassLoader.php(444): include()
#1 D:\\Projects\\foodtiger\\vendor\\composer\\ClassLoader.php(322): Composer\\Autoload\\includeFile('D:\\\\Projects\\\\foo...')
#2 [internal function]: Composer\\Autoload\\ClassLoader->loadClass('App\\\\Models\\\\MyAu...')
#3 [internal function]: spl_autoload_call('App\\\\Models\\\\MyAu...')
#4 Command line code(1): class_exists('App\\\\Models\\\\MyAu...')
#5 [internal function]: {closure}('App\\\\Models\\\\MyAu...')
#6 Command line code(1): array_filter(Array, Object(Closure))
#7 {main}
  thrown at D:\\Projects\\foodtiger\\app\\Models\\MyAuth0User.php:8)
[stacktrace]
#0 {main}
"} 
[2022-01-27 15:59:07] local.ERROR: Uncaught Error: Class 'Auth0\Login\Auth0User' not found in D:\Projects\foodtiger\app\Models\MyAuth0User.php:8
Stack trace:
#0 D:\Projects\foodtiger\vendor\composer\ClassLoader.php(444): include()
#1 D:\Projects\foodtiger\vendor\composer\ClassLoader.php(322): Composer\Autoload\includeFile('D:\\Projects\\foo...')
#2 [internal function]: Composer\Autoload\ClassLoader->loadClass('App\\Models\\MyAu...')
#3 [internal function]: spl_autoload_call('App\\Models\\MyAu...')
#4 Command line code(1): class_exists('App\\Models\\MyAu...')
#5 [internal function]: {closure}('App\\Models\\MyAu...')
#6 Command line code(1): array_filter(Array, Object(Closure))
#7 {main}
  thrown {"exception":"[object] (Symfony\\Component\\ErrorHandler\\Error\\FatalError(code: 0): Uncaught Error: Class 'Auth0\\Login\\Auth0User' not found in D:\\Projects\\foodtiger\\app\\Models\\MyAuth0User.php:8
Stack trace:
#0 D:\\Projects\\foodtiger\\vendor\\composer\\ClassLoader.php(444): include()
#1 D:\\Projects\\foodtiger\\vendor\\composer\\ClassLoader.php(322): Composer\\Autoload\\includeFile('D:\\\\Projects\\\\foo...')
#2 [internal function]: Composer\\Autoload\\ClassLoader->loadClass('App\\\\Models\\\\MyAu...')
#3 [internal function]: spl_autoload_call('App\\\\Models\\\\MyAu...')
#4 Command line code(1): class_exists('App\\\\Models\\\\MyAu...')
#5 [internal function]: {closure}('App\\\\Models\\\\MyAu...')
#6 Command line code(1): array_filter(Array, Object(Closure))
#7 {main}
  thrown at D:\\Projects\\foodtiger\\app\\Models\\MyAuth0User.php:8)
[stacktrace]
#0 {main}
"} 
